### DISCUSSION TITLE
Possibility to disable graphiQL client?

### DISCUSSION THREAD

**Discussion by peterbierman** [Netflix/dgs-framework#257] [Q&A] [answered]
when starting the DGS service the graphiQL client is default available at http://localhost:8080/graphiql.
In some environments we don't want this client active, is there a simple way to disable the client?

**Comment by paulbakker** [comment, marked as answer]
Yes, there is a `dgs.graphql.graphiql.enabled` setting available. Set it to `false` to disable.
You would typically put this in your `application.properties` or `application.yml`.

**Reply by shiyouping** [reply]
May I know where I can find the list of properties I can set in the configuration files? Your official document doesn't mention anything about this property, which is really bad.

**Reply by berngp** [reply]
We don't have that full list documented yet. Will update this thread as soon as that is the case.
